nestkernel/nest_time.cpp

```diff
@@ -77,7 +78,7 @@ Time::compute_max()
   const tic_t tmax = std::numeric_limits< tic_t >::max();
 
   tic_t tics;
-  if ( lmax < tmax / Range::TICS_PER_STEP ) // step size is limiting factor
+  if ( lmax < tmax * Range::TICS_PER_STEP_INV ) // step size is limiting factor
```

Comment:
@apeyser This does not matter to performance here, but isn't it a good idea to do calculations consistent all over?